### PR TITLE
[ENH] improve output for reusability

### PR DESCRIPTION
**Pull request by Remi-Gau**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,13 +27,79 @@ an fMRI) experiment and hard to measure.
...
+ bottom right: 1481 648
+
+Field of view in DEGREES OF VISUAL ANGLES:
+ width: 486
+ height: 686
+
+
+
+--------------------------------------------------------------------------------
+SET UP TO COPY
```
Comment (Remi-Gau) [inline README.md:93]: This below gives an example of what to copy paste for other experiments

File: README.md [hunk 2]
```diff
@@ -30,13 +27,79 @@ an fMRI) experiment and hard to measure.
...
+
+Field of view in PIXEL:
+ width: 486
+ height: 686
+
+
+
+--------------------------------------------------------------------------------
+Field of view in DEGREES OF VISUAL ANGLES:
+ top left: 795 162
```
Comment (marcobarilari) [inline README.md]: these are still in pixel, right?

Comment (Remi-Gau) [reply README.md]: should be fixed now

**Comment by marcobarilari** [comment]
LGTM

**Review by marcobarilari — APPROVED** [review]

**Pull request merged**